How-to-use-Download-url.md

```diff
@@ -11,35 +12,37 @@ You do need a separate webserver with a vhost configured to make the shows avail
 <span class="section">'''Apache'''</span>
 Example configuration if you use apache, assuming your shows are stored in /home/user/shows :
 
-  <VirtualHost "shows.domain.tld:80">
-    DocumentRoot /home/user/shows
-    ServerName shows.domain.tld
-    CustomLog /var/log/apache2/shows_access.log combined
-  
+<VirtualHost "shows.domain.tld:80">
+DocumentRoot /home/user/shows
+ServerName shows.domain.tld
+CustomLog /var/log/apache2/shows_access.log combined
+
     <Directory "/home/user/shows">
        Order deny,allow
        Allow from all
     </Directory>
-  </VirtualHost>
 
+  </VirtualHost>
 
 <span class="section">'''nginx'''</span>
 Example configuration for nginx :
-  server
```

Comment:
@BKSteve this should be in a code block

Reply:
Just a move from old one.

Reply:
Yeah just noticing things we can fix, I didn't think you messed it up lol